app/src/main/java/org/oppia/android/app/recyclerview/BindableAdapter.kt

```diff
@@ -95,63 +96,15 @@ class BindableAdapter<T : Any> internal constructor(
     internal abstract fun bind(data: T)
   }
 
-  /**
-   * The base builder for [BindableAdapter]. This class should not be used directly--use either
-   * [SingleTypeBuilder] or [MultiTypeBuilder] instead.
-   */
-  abstract class BaseBuilder<BuilderType : Any> {
```

Comment:
Doesn't ``BindableAdapterTest`` need to be updated?

Reply:
Updated BindableAdapterTest but getting some errors.